tests/test_bond_order_system.py

```diff
@@ -2,12 +2,20 @@
 import unittest
 from context import dpdata
 import glob
-from rdkit import Chem
-from rdkit.Chem import AllChem
+try:
+    from rdkit import Chem
+    from rdkit.Chem import AllChem
+    from dpdata import BondOrderSystem
+    skip_bond_order_system = False
+except ImportError:
+    skip_bond_order_system = True
```

Comment:
`skip_bond_order_system = False` can be put outside the `try...except` block.
```suggestion
try:
    from rdkit import Chem
    from rdkit.Chem import AllChem
    from dpdata import BondOrderSystem
except ImportError:
    skip_bond_order_system = True
else:
    skip_bond_order_system = False
```